twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.tsx

```diff
@@ -38,23 +39,29 @@ export default (props: Props) => {
       context: null,
       hasTabs: props.visibility !== 'direct' && !props.app,
     });
+
+    WindowState.setSuffix(props.name);
   }
 
+  const onClickChannel = () => {
+    props.id &&
+      MainViewService.select(props.id, {
+        collection: props.collection,
+        app: props.app || 'messages',
+        context: null,
+        hasTabs: props.visibility !== 'direct' && !props.app,
+      });
+
+    WindowState.setSuffix(props.name);
```

Comment:
Don't you think it is a duplicate ? We do it line 43 when channel load just after we click on it. For me this one should be removed.

Reply:
If you don't set suffix here, the title won't be updated after the first load (after login for example) or reload.

https://user-images.githubusercontent.com/36481167/123264272-26a21100-d4fa-11eb-9ba1-12f5e7f4175a.mp4

Reply:
Are you sure ? I agree that the first one (line 43) must be kept, but not the one line 55